docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.8"
+services:
+  app:
```

Comment:
How are you going to pass the Database url, username and password to the app?. Please use proper envars.